Mathlib/Computability/TMToPartrec.lean

```diff
@@ -1604,7 +1604,7 @@ def trStmts₁ : Λ' → Finset Λ'
 theorem trStmts₁_trans {q q'} : q' ∈ trStmts₁ q → trStmts₁ q' ⊆ trStmts₁ q := by
   induction q with
   | move _ _ _ q q_ih => _ | clear _ _ q q_ih => _ | copy q q_ih => _ | push _ _ q q_ih => _
-  | read q q_ih => _ | succ q q_ih => _ | pred q₁ q₂ q₁_ih q₂_ih => _ | ret => _
+  | read q q_ih => _ | succ q q_ih => _ | pred q₁ q₂ q₁_ih q₂_ih => _ | ret => _ <;>
   all_goals
```

Comment:
I don't understand what's going on here. Don't you want to remove the `all_goals`, if you're adding `<;>`?

Reply:
Good catch!  I was convinced that I had tried and it failed, but it seems to work locally now.